cmd/cortex/Dockerfile

```diff
@@ -1,7 +1,22 @@
 FROM       alpine:3.13
-RUN        apk add --no-cache ca-certificates
+RUN        apk add --no-cache ca-certificates libcap
+
+# create cortex user to run cortex as non-root
+RUN addgroup -g 10000 -S cortex && \
+    adduser  -u 10000 -S cortex -G cortex
+
+RUN mkdir /data && \
```

Comment:
Should this be `mkdir -p /data && chown -R cortex:cortex /data` ? I'm unclear when this would run and if we have to worry about migrating existing data / files to the new owner

Reply:
In the first step I just want an image that runs still root by default, but that supports running as non-root. 

That run statement just makes sure that if you decide to run a container with `--user cortex:cortex` and nothing mounted in `/data`, it has write permissions for `cortex:cortex`

Reply:
Sure, but if `/data` already exists the command will fail

Reply:
`RUN` is only run at `docker build` time, so as long as `/data` doesn't exist in the base image `alpine:3.13` we should be fine.